### PR TITLE
Function to encrypt ROT13

### PR DESCRIPTION
**Pull request by beny201**
Saw others solutions at ends, I think I overcomplicate it :) Didn't want to do by dictionary at beginning but i think it is easiest way here.

### CODE REVIEW

File: 22_kata.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            if char.isupper():
+                char_range = chr_range["upper"]
+            else:
+                char_range = chr_range["lower"]
+
+            unicode_char = ord(char)
+
+            if unicode_char + 13 > char_range[1]:
+                unicode_char = char_range[0] + \
+                    (unicode_char + 12) - char_range[1]
```
Comment (pawelzagorski) [inline 22_kata.py:23]: Did VSC format this line like that? It doesn't seem to be super big and hence I'm surprised why it got broke into two lines :)

File: 22_kata.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                coded_char = chr(unicode_char)
+                coded_sentence += coded_char
+            else:
+                coded_char = chr(unicode_char+13)
```
Comment (pawelzagorski) [inline 22_kata.py]: There are no white spaces around the mathematical operator ;)

**Review by pawelzagorski — APPROVED** [review]
It might be a bit overcomplicated as you stated, but the most important aspect is that it works as expected. Congratulations on solving this Kata! 

I added two minor comments, but they are really small. One is a question and the other is more of a reminder for the future ;)

File: 22_kata.py (hunk 1 above)
Comment (beny201) [reply 22_kata.py:23]: Yep, during saving moved it down. I don't fight with it. Btw 79 char at line it is common used or it is dead law?

File: 22_kata.py (hunk 2 above)
Comment (beny201) [reply 22_kata.py]: sharp eye :)

**Pull request merged**